lib/package.json

```diff
@@ -25,6 +25,8 @@
     "tsc-clean": "tsc --build --clean tsconfig.json"
   },
   "dependencies": {
+    "@ethersproject/constants": "^5.7.1",
+    "@ethersproject/contracts": "^5.7.1",
```

Comment:
These versions don't seem to exist:
https://www.npmjs.com/package/@ethersproject/contract?activeTab=versions
https://www.npmjs.com/package/@ethersproject/constants?activeTab=versions

Reply:
Not really sure how to interpret ethers versioning: https://www.npmjs.com/org/ethersproject

Looks like they're keeping major and feature versions in sync, but not patch versions.  A version `5.7.1` exists, but not for these particular packages.

Reply:
Fixed in #176